README.md

```diff
@@ -649,12 +649,7 @@ This section provides a minimal list of tools and libraries along with their rec
 run this pipeline outside Google Shell.
 
 1. Install Java using [OpenJDK](https://openjdk.org/) (recommended version: 19.0.1 and above).
-1. Gradle - Use the bundled [gradle wrapper to install or update](https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper) the required Gradle version automatically.
-    1. To update the Gradle version (recommended version: 8.3):
-    ```commandline
-    gradle wrapper --gradle-version <updated_version>
-    ```
-    2. Build with the updated version:
+1. Use Gradle wrapper to build the project:
```

Comment:
Gradle Wrapper

(They capitalize Wrapper in their docs so I think that's the proper name of the feature.)